ecommerce-webhosting/terraform/load_balancing.tf

```
@@ -0,0 +1,124 @@
+  prefix      = "terraform-static-content-"
+  byte_length = 2
+}
+
+module "gce-lb-http" {
+  source  = "GoogleCloudPlatform/lb-http/google"
+  version = "~> 6.2"
+
+  project           = var.project_id
+  name              = "${local.application_name}-http-lb"
+  target_tags       = ["allow-hc"]
+  firewall_networks = [module.vpc.network_name]
+
+  backends = {
+    default = {
+      description             = null
+      protocol                = "HTTP"
+      port                    = 80
+      port_name               = "http"
+      timeout_sec             = 10
+      enable_cdn              = false
+      custom_request_headers  = null
+      custom_response_headers = null
+      security_policy         = null
+
+      connection_draining_timeout_sec = null
+      session_affinity                = null
+      affinity_cookie_ttl_sec         = null
+
+      log_config = {
+        enable      = true
+        sample_rate = 1.0
+      }
+
+      health_check = {
+        check_interval_sec  = null
+        timeout_sec         = null
+        healthy_threshold   = null
+        unhealthy_threshold = null
+        request_path        = "/"
+        port                = 80
+        host                = null
+        logging             = null
+      }
+
+      groups = [
+        {
+          # Each node pool instance group should be added to the backend.
+          group                        = module.mig.instance_group
+          balancing_mode               = null
+          capacity_scaler              = null
+          description                  = null
+          max_connections              = null
+          max_connections_per_instance = null
+          max_connections_per_endpoint = null
+          max_rate                     = null
+          max_rate_per_instance        = null
+          max_rate_per_endpoint        = null
+          max_utilization              = null
+        },
+      ]
+
+      iap_config = {
+        enable               = false
+        oauth2_client_id     = null
+        oauth2_client_secret = null
+      }
+    }
+  }
+}
+
+resource "google_compute_url_map" "ml-bkd-ml-mig-bckt-s-lb" {
+  // note that this is the name of the load balancer
+  name            = var.network_name
+  default_service = module.gce-lb-http.backend_services["default"].self_link
+
+  host_rule {
+    hosts        = ["*"]
+    path_matcher = "allpaths"
+  }
+
+  path_matcher {
+    name            = "allpaths"
+    default_service = module.gce-lb-http.backend_services["default"].self_link
+
+    path_rule {
+      paths = [
+        "/api",
+        "/api/*"
+      ]
+      service = module.gce-lb-http.backend_services["default"].self_link
+    }
+
+    path_rule {
+      paths = [
+        "/front",
+        "/front/*"
+      ]
+      service = google_compute_backend_bucket.assets.self_link
+    }
+  }
+}
+
+resource "google_compute_backend_bucket" "assets" {
+  name        = random_id.assets-bucket.hex
```

```suggestion
  name        = "${local.application_name}-static-backend"
```
Standardize same prefix for all